packages/airnode-node/src/evm/wallet.ts

```diff
@@ -15,16 +15,17 @@ import { Config } from '../config/types';
  * 1/1st31bits/2nd31bits/3rd31bits/4th31bits/5th31bits/6th31bits.
  *
  * @param sponsorAddress A string representing a 20bytes hex address
+ * @param protocolId An optional string representing the protocol id. Defaults to '1'.
  * @returns The path derived from the address
  */
-export const deriveWalletPathFromSponsorAddress = (sponsorAddress: string): string => {
+export const deriveWalletPathFromSponsorAddress = (sponsorAddress: string, protocolId?: string) => {
```

Comment:
Nit: What do you think about using a default value instead of making `protocolId` optional? That way you wouldn't need to add the `|| '1' `below on line 28.

Reply:
It's a good idea, I'll open another PR :+1: